crates/driver/src/domain/competition/mod.rs

```diff
@@ -46,20 +45,20 @@ pub struct Competition {
     pub settlement: Mutex<Option<Settlement>>,
 }
 
+/// Solution information revealed to the protocol by the solver. Note that the
+/// calldata is never revealed, as it would allow the solution to be stolen by
+/// other solvers.
```

Comment:
**Not for this PR**, but I believe CIP-20 stipulates that solvers need to reveal calldata (well, specifically, a "solution" which can be turned into calldata). We _may_ want to change the driver co-location API to accommodate this. @harisang - what are your thoughts?